packages/zk-certificates/contracts/ZkCertificateRegistry.sol

```diff
@@ -100,60 +135,69 @@ contract KYCRecordRegistry is Initializable, IKYCRegistry {
         }
 
         // Set merkle root
-        merkleRoot = currentZero;
+        merkleRoots.push(currentZero);
         _GuardianRegistry = GuardianRegistry(GuardianRegistry_);
 
         // Set the block height at which the contract was initialized
         initBlockHeight = block.number;
     }
 
     /**
-     * @notice addZkKYCRecord issues a zkKYC record by adding it to the Merkle tree
-     * @param leafIndex - leaf position of the zkKYC in the Merkle tree
-     * @param zkKYCRecordHash - hash of the zkKYC record leaf
-     * @param merkleProof - Merkle proof of the zkKYC record leaf being free
+     * @notice addZkCertificate issues a zkCertificate record by adding it to the Merkle tree
+     * @param leafIndex - leaf position of the zkCertificate in the Merkle tree
+     * @param zkCertificateHash - hash of the zkCertificate record leaf
+     * @param merkleProof - Merkle proof of the zkCertificate record leaf being free
      */
-    function addZkKYCRecord(
+    function addZkCertificate(
```

Comment:
Can you please also rename the functions in the DevnetGuardian contract? The ABI should be the same.

Reply:
I thought that devnetGuardian is for zkKYC. Also why do we need it as why can't guardians call zkCertificateRegistry directly?

Reply:
It is for the test userflow on devnet and testnet. Because we do not have real guardians there a user also plays the guardian role and creates his own zkKYC. To not need to whitelist all users, we instead whitelist this devnetGuardian contract that everyone can use and that has the same ABI.

Reply:
I'll change it.